network/network.go

```diff
@@ -280,6 +280,20 @@ func (n *Network) Client(address string) (*PeerClient, error) {
 	return client, nil
 }
 
+// HasConnection returns true if network has a connection on a given address.
+func (n *Network) HasConnection(address string) bool {
```

Comment:
Care to rename to `ConnectionStateExists` and `ConnectionState`?

Reply:
done